docs/DevelopmentPlan/DevelopmentPlan.md

```diff
@@ -83,12 +83,42 @@ requests.
   
 # 7 Workflow Plan
 
--   How will you be using git, including branches, pull request, etc.?
+### Git Workflow
+For this project, Git Feature Branch workflow will be utilized for feature development. This strategy will enable the development of new features in isolated branches, enhancing collaboration and reducing the risk of introducing bugs in the main branch. In general branches will follow the naming convention of <type of branch>/<name of branch>. Some examples are given below:
+- `feature/<name of feature>`
+- `fix/<name of fix>`
+- `docs/<name of doc changed>`
 
--   How will you be managing issues, including template issues, issue
-    classification, etc.?
 
--   Use of CI/CD
+### Pull Requests
+Pull Requests (PRs) will be created by the collaborators of the project to merge their respective feature branches into the main branch. Upon opening a PR, a short description of what the changes are should be provided. PRs should be linked to at least one issue in the repository to ensure traceability of work. This workflow will ensure that minimal conflicts are introduced and also will reduce the probability of introducing bugs into the main branch. To merge a PR, it will need to be approved by at least one of the collaborators.
+
+### Github Project, Github Issues and Labels/Tags
+Github Projects along with Github Issues will be utilized for the management and tracking of tasks in this project. Github Projects will be used to track issues in Issues will be created with different custom predefined templates to standardize the process and save time containing the following details at a minimum:
```

Comment:
Can you elaborate on how github projects will be used here for example what todo, in progress, in review, done mean as well as incorporating milestone tracking